_pages/en/strategy-build-vs-buy.md

```diff
@@ -67,48 +67,60 @@ All ESDC personnel involved in the exploration of IT Solutions and in IT investm
 
 Moving to the digital age requires improving IT's responsiveness and promoting a shared accountability with ESDC stakeholders on the use of technology.
 
-Yet, the current principle when obtaining IT Solutions is to "Buy what we can, Build what we must."
+Yet, the current principle when acquiring IT Solutions is to "Buy what we can, Build what we must."
 The assumed reasoning behind this overarching principle is that by purchasing an IT Solution from a third party, risks and responsibilities of its operations are handled by a company who has more resources and technical expertise to produce and maintain software.
 
 This principle unfortunately is leading to an oversimplified approach that doesn't take into account that, as a government department, ESDC is subject to multiple and fast-paced changes, whether at the [legal, political, organizational or technological levels](https://www.canada.ca/en/employment-social-development/corporate/reports/departmental-plan/2020-2021.html#h2.2-3.2), that impacts its ability to deliver on its mandate.
```

Comment:
I'd remove 'unfortunately' as I don't think its needed. Or if keeping, perhaps put as "this principle is unfortunately leading....'. If staying as is, think commas might be required around unfortunately.